### ISSUE TITLE
"[FTL] Could not create runner:"

### ISSUE THREAD

**Issue by aleksey-vi** [projectdiscovery/nuclei#5085]
### Nuclei version:
Nuclei v3.2.4

### Steps To Reproduce:

Example: steps to reproduce the behavior:
1. Run 'nuclei -l project_file.burp -im burp'
2. See error...

![image](https://github.com/projectdiscovery/nuclei/assets/65017000/0452b9e6-9d56-46e5-a6a3-3117c2a2f424)

**Comment by ehsandeep** [comment]
@aleksey-vi nuclei accpets burp's exported request/respose file as input, looks like you are feeding project file as input?

**Comment by aleksey-vi** [comment]
Yes you're right I did exactly that))))) I couldn't find anything in the documentation about it.... can you tell me what format it expects the data in? i should save from burp proxy->HTTP history-> test.burp or test.xml ?

> @aleksey-vi nuclei accpets burp's exported request/respose file as input, looks like you are feeding project file as input?

**Comment by aleksey-vi** [comment]
is this normal behaviour? whatever format one chooses))))

![image](https://github.com/projectdiscovery/nuclei/assets/65017000/02a473c3-dca6-4458-9244-bfcb258ef45d)

**Comment by jimen0** [comment]
The process was killed. Dmesg should show why but I'll guess it's an OOM issue.

How large is that project file? How many targets are there in it?

**Comment by aleksey-vi** [comment]
url addresses 1000+ but the domains themselves are 3) and there are any restrictions on the number of urls ? file 425MB

> Процесс был убит. Dmesg должен показать причину, но я предполагаю, что это проблема OOM.
> 
> Насколько велик этот файл проекта? Сколько в нем целей?

**Comment by jimen0** [comment]
dmesg should show why the process was killed but I'd bet the cause was OOM.

**Comment by Mzack9999** [comment]
1000 urls shouldn't be an issue, it could be related to machine low RAM amount. Closing due to inactivity and lack of reproduction steps.
